.github/workflows/test.yml

```
@@ -0,0 +1,22 @@
+name: Run test for dmX on push to main and PR's
```

```suggestion
name: Build and test
```
Just make this a bit shorter to make the check section of the PRs a bit easier to read. The other information is not really necessary, since we know in which repository we are and in the end we don't care too much what triggered the workflow.